include/fastdds/rtps/common/Locator.h

```diff
@@ -631,10 +651,20 @@ inline std::ostream& operator <<(
         std::ostream& output,
         const LocatorList_t& locList)
 {
-    for (auto it = locList.m_locators.begin(); it != locList.m_locators.end(); ++it)
+    output << "[";
+    if (locList.size() > 0)
     {
-        output << *it << ",";
+        output << *(locList.begin()) ;
```

Comment:
Why do you print the first element separately? I think it is clearer and easier to read and understand to print all the elements of the list within the loop